addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -448,8 +450,14 @@ export class Wysiwyg extends Component {
             getPowerboxElement: () => {
                 const selection = (this.options.document || document).getSelection();
                 if (selection.isCollapsed && selection.rangeCount) {
-                    const baseNode = closestElement(selection.anchorNode, 'P:not([t-field]), DIV:not([t-field]):not(.o_not_editable)');
-                    const fieldContainer = closestElement(selection.anchorNode, '[data-oe-field]');
+                    const deepestNode = getDeepestPosition(selection.anchorNode, selection.anchorOffset);
```

Comment:
The `deepestNode` variable doesn't contain a node. Do this instead:
```js
const [deepestNode] = ...
```
By doing that you won't need to do `[0]` later on.

Reply:
@dmo-odoo I have implemented the requested changes.